matrix_sdk/Cargo.toml

```diff
@@ -68,6 +68,7 @@ features = ["wasm-bindgen"]
 
 [dev-dependencies]
 async-trait = "0.1.40"
+async-std = { version = "*", features = ["unstable"] }
```

Comment:
Is this really meant to be `version = "*"`?

Reply:
Indeed, no.